varats/varats/plots/blame_interaction_degree.py

```diff
@@ -1451,6 +1435,29 @@ def calc_missing_revisions(
         )
 
 
+class BlameInteractionDegreeMultiLibGenerator(
+    PlotGenerator,
+    generator_name="interaction-degree-multi-lib-plot",
+    options=[
+        REQUIRE_REPORT_TYPE, REQUIRE_MULTI_CASE_STUDY, REQUIRE_BASE_LIB,
+        REQUIRE_INTER_LIB, OPTIONAL_SHOW_CHURN, OPTIONAL_EDGE_COLOR,
+        OPTIONAL_COLORMAP, OPTIONAL_SHOW_CVE, OPTIONAL_SHOW_BUGS,
+        OPTIONAL_CVE_LINE_WIDTH, OPTIONAL_BUG_LINE_WIDTH, OPTIONAL_CVE_COLOR,
+        OPTIONAL_BUG_COLOR, OPTIONAL_VERTICAL_ALIGNMENT
+    ]
+):
+    """Generates multi-lib degree plot(s) for the selected case study(ies)."""
+
+    def generate(self) -> tp.List[Plot]:
+        return [
+            BlameInteractionDegreeMultiLib(
+                self.plot_config, **{
+                    **self.plot_kwargs, "case_study": cs
+                }
+            ) for cs in self.plot_kwargs["case_study"]
```

Comment:
I think this can be simplified:

```suggestion
            BlameInteractionDegreeMultiLib(
                self.plot_config, case_study=cs, **plot_kwargs
            ) for cs in self.plot_kwargs["case_study"]
```

Reply:
Initially I tried the same, but this fails with:
`TypeError:  varats.plots.blame_interaction_degree.BlameInteractionDegreeMultiLib() got multiple values for keyword argument 'case_study'`

Reply:
Then I suggest deleting the entry from the kwargs before. This makes it also more obvious that it is overridden.